package.json

```diff
@@ -13,7 +13,7 @@
     "lint": "prettier --check \"src/**/*.ts\" \"test/**/*.ts\" \"docs/*.md\" *.md package.json tsconfig.json --end-of-line auto",
     "lint:fix": "prettier --write \"src/**/*.ts\" \"test/**/*.ts\" \"docs/*.md\" *.md package.json tsconfig.json --end-of-line auto",
     "pretest": "tsc --noEmit -p test",
-    "test": "jest",
+    "test": "node --experimental-vm-modules node_modules/jest/bin/jest.js",
```

Comment:
```suggestion
    "test": "npm run build && node --experimental-vm-modules node_modules/jest/bin/jest.js",
```

Reply:
this should fix the failing e2e test.

Reply:
In CI, a build is run anyway before a test. The test failure seems to be another issue that I'm experimenting with now.

Reply:
We should anyway add it. Because it fails locally